arbeitszeit_web/get_statistics.py

```diff
@@ -1,36 +1,48 @@
 from dataclasses import dataclass
+from decimal import Decimal
 
 from arbeitszeit.use_cases import StatisticsResponse
+from arbeitszeit_web.colors import Colors
+from arbeitszeit_web.plotter import Plotter
 from arbeitszeit_web.translator import Translator
 
 
 @dataclass
 class GetStatisticsViewModel:
     registered_companies_count: str
     registered_members_count: str
+    cooperations_count: str
+    certificates_count: str
+    available_product: str
     active_plans_count: str
     active_plans_public_count: str
     average_timeframe_days: str
     planned_work_hours: str
     planned_resources_hours: str
     planned_means_hours: str
 
+    barplot_certificates: str
+    barplot_means_of_production: str
+    barplot_plans: str
+
 
 @dataclass
 class GetStatisticsPresenter:
-    translator: Translator
+    trans: Translator
```

Comment:
Why did you shorten the name `translator`? When reading this I think less work on the part of the reader is required with the non-shortened version.